### ISSUE TITLE
A11y_AzureToolsforVSCode_AzureFunction_Selectalocationfornewresources_ScreenReader: The NVDA is not announcing the expand/collapsed state for the "Recommended" and "Other" list items.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode-azurefunctions#2943]
### Actual:
When we navigate to "Recommended" and "Other" list items, the NVDA announces "chevron-down Recommended  1 of 30" for "Recommended" list item and "chevron-down Other  22 of 30" for "Other" list item but it does not announce the state of the control. 
This controls can be expanded or collapsed ,for this NVDA announces "chevron-down" if the control is in expanded state and "chevron-right" if the control is in collapsed state.

**This issue is also observed with JAWS.**

### Expected:
The NVDA should announce the state of the control descriptively for example here it should announce like "Recommended  1 of 30 expanded " and "Other 2 of 30 expanded".

**GitHubTags:**#A11yMAS;#A11yTCS;BM_AzureToolsforVSCode_Win32_Aug2021;#Azure Tools for VS
Code;#A11ySev2;#WCAG1.3.1;#DesktopApp;# Win10;#NVDA;#JAWS;#Element:Header;

### Environment Details:
Visual Studio Code
Azure Function
Operating System: Windows 10 Pro 21H1
Screen Reader: NVDA(2021.1),JAWS(2021.2107.12 ILM)

### Steps to Reproduce:

1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. "Azure Function" screen should open. TAB to "Install" button and hit ENTER key to install it.
4. "Azure" button will appear in navigation pane. TAB to it and select it.
5. Start NVDA.
6. TAB to any Subscription and expand options using Shift+F10 then TAB to "Create Function App in Azure" control and select it.
7. Edit some name for new function and hit ENTER key, then select "Powershell 7.0" as a "Runtime stack" and hit ENTER key.
8. "Select a location for new resources" edit control will appear. TAB to "Recommended" list item.
9. Observe the announcement by screen reader for the "Recommended" list item.

### User Impact:
The screen reader users won't be able to understand the control can be collapsed or expanded.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by ejizba** [comment]
The existing grouping is a custom implementation for our extension. We want to use VS Code's grouping, which should be fully accessible, but unfortunately it's not exposed to extensions yet. Tracking issue here: https://github.com/microsoft/vscode/issues/74967

**Comment by 12-shweta** [comment]
We will track this bug using provided tracking id https://github.com/microsoft/vscode/issues/74967

**Comment by 12-shweta** [comment]
@ejizba The tracking issue has been closed but when we are checking on the Visual studio code we are still able to reproduce the issue, could you please provide build where the fixes will be available:

**Comment by jinglouMSFT** [comment]
@12-shweta The fix has been merged. Please verify. Thanks.

**Comment by 12-shweta** [comment]
The issue is still reproducible that is NVDA is not announcing the expand/collapsed state for the "Recommended" and "Other" list items.
![image](https://user-images.githubusercontent.com/54656883/172593398-23c01145-3295-412e-87f8-9c25220ce01a.png)

**Comment by MicroFish91** [comment]
Ahh, I need to update the dependency version number to pick up the new azuretools package changes that were made.  Update coming soon!

**Comment by 12-shweta** [comment]
Ok. Once update is done do let me know, i will regress this issue.

**Comment by MicroFish91** [comment]
Updated!  https://github.com/microsoft/vscode-azurefunctions/pull/3215

version:  1.7.4-alpha.3

**Comment by nturinski** [comment]
https://dev.azure.com/ms-azuretools/534e1e1c-b364-44d0-b5ee-dce879a5106e/_apis/build/builds/48035/artifacts?artifactName=vsix&api-version=7.0&%24format=zip

**Comment by jinglouMSFT** [comment]
@12-shweta Can you please verify and regress the fix?  Thanks.

**Comment by 12-shweta** [comment]
Verified the bug in the mentioned version. Fix has been provided properly as mentioned in external bug.

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;

**Comment by InduPriya1805** [comment]
GitHubTags:#BM_AzureToolsforVSCode_Win32_Aug2021;
